EMML/8001-9000/EMML8098.xml

```diff
@@ -41,8 +41,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                <msContents>
                   <summary/>
                   <msItem xml:id="ms_i1">
-                     <locus from="1" to="35r"></locus>
                      <title ref="LIT1340EnochE">Henok</title>
+                     <textLang mainLang="gez"/>
```

Comment:
for EMML manuscripts when we edit them we should when we can also always edit institutions, just as vHMML has done. 
See e.g. EMML6260 how this can be done.